src/main/java/net/william278/velocitab/Velocitab.java

```diff
@@ -170,6 +173,14 @@ private void disableScoreboardManager() {
         }
     }
 
+    private void prepareSortingManager() {
+        this.sortingManager = new SortingManager(this);
```

Comment:
Probably no need to instantiate this class if sorting is disabled, right?

Reply:
I would keep it, so if someone reloads the plugin it's ready to use.